libraries/AP_HAL_ChibiOS/hwdef/ElfinH7-Wing/README.md

```diff
@@ -0,0 +1,84 @@
+# ElfinH7-Wing Flight Controller
+
+## Features
+
+ - MCU - STM32H750 32-bit processor running at 480 MHz
+ - 16MByte Serial NOR flash via QuadSPI
+ - Barometer - spl06-001 on I2C1 bus
+ - IMUs - 1x icm-42688-p, 1x icm-20608-g
+
+## Interface
+
+ - 7x UARTs (1,2,3,4,6,7,8)
+ - 13x PWM Outputs (12 Motor Output, 1 LED)
+ - 1x RC input PWM/PPM, SBUS
+ - 2x I2C port for external compass, airspeed sensor, etc.
+ - SPI4 port
+ - USB port (with remote cabling)
+ - CAN port
+ - External Buzzer
+ - Dual Switchable Camera inputs
+ - Built-in OSD
+- microSD card
+- Battery input voltage: 2S-6S
+ - BEC: 1x 5V/2A, 1x 9/12V/2A, 1x 5/6/7.2V/12A
```

Comment:
should mention how to control voltage selection and power control down lower in readme